odl/contrib/fom/supervised.py

```diff
@@ -130,7 +130,8 @@ def mean_absolute_error(data, ground_truth, mask=None, normalized=False):
     return fom
 
 
-def mean_value_difference(data, ground_truth, mask=None, normalized=False):
+def mean_value_difference(data, ground_truth, mask=None, normalized=False,
+                          force_lower_is_better=False):
```

Comment:
The flag should be added to the doc as well, describing what it does.

Reply:
Also, even if the flag does not change anything I think it is good to have a comment in the code that states this. See, .e.g., how @sbanert did [here](https://github.com/sbanert/odl/blob/952db84246ac131648aa09523239160e5a874110/odl/contrib/fom/supervised.py#L82-L84).